core/modules/qproc/QuerySession.h

```diff
@@ -154,10 +163,10 @@ class QuerySession {
     void _applyLogicPlugins();
     void _generateConcrete();
     void _applyConcretePlugins();
-    std::string toString(); // Debug
 
     // Iterator help
     std::vector<std::string> _buildChunkQueries(ChunkSpec const& s) const;
+    void addChunk(const ChunkSpecVector& cs);
```

Comment:
Why did you make it private, the implementation was removed completely, I'd expect that declaration should disappear too?